front-end/src/components/registrationform.js

```diff
@@ -19,8 +20,29 @@ class RegistrationForm extends Component {
     console.log("The input values are", Object.values(this.state));
     // make axios call to backend registration route
 
+    const newUser = {
+      first_name: this.state.first_name,
+      last_name: this.state.last_name,
+      email: this.state.email,
+      password: this.state.password
+    }
+
+    Axios.post("http://localhost:5000/signup", newUser)
+    .then(response => {
+      console.log(response)
+      alert("Success")
+      window.location.reload()
```

Comment:
instead of reload, let's do
```
 this.props.history.push('/navbar')
```
this redirects to whatever page we want to go to after submit.  In this case, the uploads page, but we don't have that yet, so go to whatever route you want just to test.